content/blog/2019/07/2019-07-24-azure-artifact-manager.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Managing Jenkins Artifacts with the Azure Artifact Manager Plugin"
+tags:
+- general
+- azure
+- plugin
+author: jshen
+---
+
+Jenkins stores all generated artifacts on the master server filesystem. This presents a couple of challenges especially when you try to run Jenkins in the cloud:
+
+* As the number of artifacts grow, your Jenkins master will run out of disk space. Eventually, performance can be impacted.
+* Frequent transfer of files between agents and master may cause load, CPU or network issues which are always hard to diagnose.
+
+Several existing plugins allow you to manage your artifacts externally. To use these plugins, you need to know how they work and perform specific steps in your job's configuration. And if you are new to Jenkins, you may find it hard to follow existing samples in Jenkins tutorial like https://jenkins.io/doc/pipeline/tour/tests-and-artifacts/[Recording tests and artifacts].
+
+So, if you are running Jenkins in Azure, you can consider automatically managing new artifacts on https://azure.microsoft.com/en-us/services/storage/[Azure Storage]. The new https://plugins.jenkins.io/azure-artifact-manager[Azure Artifact Management plugin] allows you to store artifacts in Azure blob storage and simplify your existing Jenkins jobs that contain Jenkins general artifacts management steps. This approach will give you all the advantages of a cloud storage, with less effort on your part to maintain your Jenkins instance.
+
+== Configuration
+
+======  Azure storage account
```

Comment:
Heading levels are not aligned here. It is unlikely a root cause, but who knows

Reply:
```suggestion
===  Azure storage account
```
the level 5 headings should be all level 2 (level 0 = page title). otherwise asciidoctor prints a `WARNING` message. The build script will fail the build if it contains any `WARNING`. 

Note: the check is case sensitive, so `warning`s are ignored (also the only current `warning` is related to outdated version of a library, nothing to do with this PR, can be ignored).